src/models/PluginSettingsTab.ts

```diff
@@ -80,7 +82,16 @@ export class PluginSettingsTab extends PluginSettingTab {
 					this.plugin.settings.autoCreateIndexFile = value
 					await this.plugin.saveSettings()
 				}))
-
+		
+		new Setting(containerEl)
+			.setName("Auto include preview")
+			.setDesc("This will automatically include previews when creating index files (!) ")
+			.addToggle((component) => component.setValue(this.plugin.settings.includeFileContent)
+				.onChange((value) => __async(this, null, function* () {
+					this.plugin.settings.includeFileContent = value;
+					yield this.plugin.saveSettings();
+			})));
+		
```

Comment:
Please change this to use the async & await keywords otherwise the CI won't be happy with this way of doing it.

Reply:
Aah, yes. Done. Thanks.